src/Engine/StateTrackers/BuildCountsTracker.java

```diff
@@ -32,7 +32,7 @@ public GameEventQueue receiveCreateUnitEvent(Unit unit)
     return null;
   }
   @Override
-  public GameEventQueue receiveTurnInitEvent(Army army, int turn)
+  public GameEventQueue receiveTurnEndEvent(Army army, int turn)
```

Comment:
Why switch this (and `TransformationTracker`) to work on end turn instead of init turn? Won't this increase the likelihood of having stale info, since stuff could change between the end of one turn and the beginning of the next?

Reply:
I'm not sure how a CO could end up building units when it's not his turn.
I guess I could see a CO that transforms *other* people's units, though that's kinda weird.
As a result, I'm not sure the stale data question is all that relevant.

Truth be told, the main reason I did it was to clean up Cinder's build count numbers so they don't show up during her next turn-init cycle.
There's probably a better way of doing that than throwing the data away early, but I haven't really thought about it and we don't currently have any cross-CO usecases for these trackers.